src/analysis/normed/order/basic.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Anatole Dedecker, Yaël Dillies
 -/
 import algebra.order.group.type_tags
+import analysis.normed.order.lattice
```

Comment:
This import is the wrong way around.

Reply:
Do you mean that `analysis.normed.order.lattice` should import `analysis.normed.order.basic` and not the other way around? That's probably true but then I'll have to move all the results about `normed_lattice_field` (which is probably not such a big thing...)

Reply:
Yes, exactly.